backend/src/v4/models/chatEvent.js

```diff
@@ -16,11 +16,17 @@
  */
 
 "use strict";
+const archiver = require("archiver");
+const fs = require("fs");
 const { v5Path } = require("../../interop");
 const EventsManager = require(`${v5Path}/services/eventsManager/eventsManager`);
 const EventsV5 = require(`${v5Path}/services/eventsManager/eventsManager.constants`).events;
+const Mailer = require("../mailer/mailer");
```

Comment:
would also be safer if you just import the function instead of the whole mailer

(It was probably done inline with the code due to circular dependencies...)